docs/projects/project_based/template/index.rst

```diff
@@ -0,0 +1,189 @@
+   +-----------------------+----------+------------+--------------+--------+
+   | cfg_cm1_sdi2_cz2_ddr0 | 1        | 2          | 2            | 0      |
+   +-----------------------+----------+------------+--------------+--------+
+   | cfg_cm1_sdi2_cz2_ddr1 | 1        | 2          | 2            | 1      |
+   +-----------------------+----------+------------+--------------+--------+ 
+   | cfg_cm1_sdi4_cz2_ddr0 | 1        | 4          | 2            | 0      |
+   +-----------------------+----------+------------+--------------+--------+ 
+   | cfg_cm1_sdi4_cz2_ddr1 | 1        | 4          | 2            | 1      |
+   +-----------------------+----------+------------+--------------+--------+ 
+   | cfg_cm1_sdi8_cz2_ddr0 | 1        | 8          | 2            | 0      |
+   +-----------------------+----------+------------+--------------+--------+ 
+   | cfg_cm1_sdi8_cz2_ddr1 | 1        | 8          | 2            | 1      |
+   +-----------------------+----------+------------+--------------+--------+
+   
+Tests
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+\**\* MENTION IF ANY MODES ARE AVAILABLE FOR TESTS \**\*
+
+The following test program files are available:
+
+=============== ==========================================
+Test program    Usage
+=============== ==========================================
+test_program_si Tests the parallel interface capabilities.
+test_program_pi Tests the serial interface capabilities.  
+=============== ==========================================
+
+Available configurations & tests combinations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+============= =============== ===================================
+Configuration Test            Build command 
+============= =============== ===================================
+cfg_si        test_program_si make CFG=cfg_si TST=test_program_si
+cfg_pi        test_program_pi make CFG=cfg_pi TST=test_program_pi 
+============= =============== ===================================
+
+.. warning::
+    Mixing a wrong pair of CFG and TST will result in a building errror. 
+    Please checkout the proposed combinations before running a custom test.
+    
+Clock scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Building the test bench
+-------------------------------------------------------------------------------
+
+The testbench is built upon ADI's generic HDL reference design framework.
+ADI does not distribute compiled files of these projects so they must be built
+from the sources available :git-hdl:`here </>` and :git-testbenches:`here </>`,
+with the specified hierarchy described :ref:`build_tb/set_up_tb_repo </>``.
```

Comment:
:ref:`build_tb/set_up_tb_repo </> doesn't seem to give me a link. Is this the right syntax to bring us to the right page when it goes mainstream? If so, ignore this comment.